ci2/ci/build.py

```diff
@@ -658,8 +660,17 @@ def config(self, deploy):  # pylint: disable=unused-argument
                                           parent_ids=self.deps_parent_ids())
 
     async def cleanup(self, batch, deploy, sink):
-        # namespace cleanup will handle deployments
-        pass
+        script = ''
+        for tag in self.log:
+            script += f'kubectl -n {self.namespace} logs -l app={tag}\n'
+        script += 'date\n'
+        self.job = await batch.create_job(CI_UTILS_IMAGE,
+                                          command=['bash', '-c', script],
```

Comment:
This looks fine to me but where can I see the logs on the ui page?

Reply:
They will be in the logs of a job called, for example, "deploy_batch_logs", which is an always run dependency of dependency of the sink

Reply:
I'm still not seeing it. Can you confirm you get the output you want or give me a link?